ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -82,6 +82,20 @@ interface Browser {
     prettyUserAgent: string;
 }
 
+type WindowSelector = unknown;
+
+interface SwitchToWindowPredicateData {
```

Comment:
I propose to rename this interface as `SwithToWindowPredicate`.

Reply:
`SwitchToWindowPredicate` is a function, this type represents its argument.

Reply:
We can use `SwitchToWindowPredicateArgument` instead.

Reply:
SwitchToWindowPredicateArgs?

Reply:
I tried to search the `PredicateData`, `PredicateArgument` and `PredicateArgs` names in https://github.com/DefinitelyTyped/DefinitelyTyped repository and I did not find such names.
The predicate data argument has a name that describes the entity passed there.

So, I propose the `WindowData` or `WindowPartialData`.

Reply:
`Args` in different forms used in DefinitelyTyped:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5344bfc80508c53a23dae37b860fb0c905ff7b24/types/wampy/index.d.ts#L15

Reply:
`WindowData` is a bit obscure. What do you think about `WindowFilterData`?

Reply:
Ok, let it be.